data/blog/posts/2022/2022-02-01-how-to-migrate-legacy-php-applications-without-stopping-development-of-new-features.md

```diff
@@ -7,32 +7,36 @@ perex: |
 ---
 
+*This is an introduction post by [rajyan](https://twitter.com/unagiunag), who used Rector to migrate an extensive PHP application from FuelPHP to Laravel.*
+
 ## Difficulty of Migrating Legacy PHP Applications
 
-Migrating requires high knowledge of PHP, a deep understanding of the application, **and time** especially in applications running in production. Migration takes time and man-hours for code modification and testing that the application works after the migration.
+Migrating requires high knowledge of PHP, a deep understanding of the application, **and time**. Especially in applications running in the production. Migration takes time and man-hours for code modification and testing that the application works after the migration.
 
-While migrating is going on, you have to stop implementing new features; otherwise, you'll fall into a hell of conflict and outdated branches.
-Stopping development for a specific time for the migration would be a difficult decision.
+While migrating is going on, you have to stop implementing new features. Otherwise, you'll fall into a hell of conflict and outdated branches. Stopping development for a specific time for the migration would be a difficult decision.
 
-## How to solve it?
+## How to Solve it?
 
 Let's see the concept first:
 
-<img src="/assets/images/blog/2022/automated_migration_concept.png" alt="" style="max-width: 35em" class="img-thumbnail">
+<img src="/assets/images/blog/2022/automated_migration_concept.png" alt="" style="max-width: 35em" class="img-thumbnail mb-4">
+
+The most important thing is to avoid conflicts.
+
+We have to be careful not to edit the same file in the "Development branch" and the "Migration branch". We continue developing new features as usual in the "Development branch" directory, for example, `old/app`. Then we apply Rector rules to migrate `old/app` and copy the complete result to `new/app` in the "Migration branch", which automated the process in CI.
+
+We are free from conflicts by always editing the app code in the "Development branch" and only adding files in the migration branch. We can also make our "Migration branch" to the "Development branch" by merging them.
 
-The most important thing was to avoid conflicts.
-We were careful not to edit the same file in the “Development branch” and the “Migration branch”. We continued developing new features as usual in the “Development branch” directory, for
-example, `old/app`. Then we applied Rector rules to migrate `old/app` and copied the whole result to `new/app` in the “Migration branch”, which automated the process in CI.
-We were free from conflicts by always editing the app code in the “Development branch” and only adding files in the migration branch. We also could make our “Migration branch” to the “Development branch” by just merging them.
+<br>
```

Comment:
The past tense (e.g. "was") usually describes some experience that is way back in history.

On the other hand, the present tense ("is") is more interactive and can drag reader right now.  

I've updated few sentences to follow this technique. Let me know if it's ok :+1:

Reply:
Thank you! The changes are totally fine.